examples/build-nginx.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 set -e
 
+if [ "$EUID" -ne 0 ]
+    then
+    echo "This script must be run as root."
```

Comment:
If it was just `run` I think I'd get away with some scattered `sudo`s, but because `begin` needs to be run as root (for chrooting during the ACI extracting) every single command in here would need sudo to be able to modify the files in `.acbuild`.